src/openai/lib/azure.py

```diff
@@ -0,0 +1,210 @@
+from __future__ import annotations
+
+import os
+from typing import Mapping, overload
+
+import httpx
+
+from .._types import NOT_GIVEN, Timeout, NotGiven
+from .._utils import is_mapping
+from .._client import OpenAI, AsyncOpenAI
+from .._models import FinalRequestOptions
+from .._exceptions import OpenAIError
+from .._base_client import DEFAULT_MAX_RETRIES, BaseClient
+
+_deployments_endpoints = set(["/completions", "/chat/completions"])
```

Comment:
We can add a few more to this list :)

```suggestion
_deployments_endpoints = set(["/completions", "/chat/completions", "/embeddings", "/audio/transcriptions", "/audio/translations"])
```